### PR TITLE
Memory Writes are performed in third stage

### PR DESCRIPTION
**Pull request by Willian-Nunes**

### CODE REVIEW

**Comment by Willian-Nunes** [comment]
image
image

**Comment by aedalzotto** [comment]
@carol045 Please test this branch with coremark and berkeley COE before we approve.

File: proto/privileged/privileged.srcs/sources_1/ip/.Xil/.BRAM.xcix.lock
Comment (aedalzotto) [inline proto/privileged/privileged.srcs/sources_1/ip/.Xil/.BRAM.xcix.lock:1]: Add `**/.Xil` to .gitignore

File: proto/privileged/privileged.srcs/sources_1/ip/BRAM.xcix
Comment (aedalzotto) [inline proto/privileged/privileged.srcs/sources_1/ip/BRAM.xcix:1]: Add `*.xcix` to .gitignore

File: proto/privileged/privileged.srcs/sources_1/ip/.Xil/.BRAM.xcix.lock
Comment (aedalzotto) [reply proto/privileged/privileged.srcs/sources_1/ip/.Xil/.BRAM.xcix.lock:1]: First commit this folder removal

File: proto/privileged/privileged.srcs/sources_1/ip/BRAM.xcix
Comment (aedalzotto) [reply proto/privileged/privileged.srcs/sources_1/ip/BRAM.xcix:1]: First commit this file removal

**Review by aedalzotto — CHANGES_REQUESTED** [review]
Just remove temporary and binary files, commit the removal, add them to .gitignore and it's good to go!

**Review by aedalzotto — APPROVED** [review]

**Pull request merged**